sonic_package_manager/service_creator/creator.py

```diff
@@ -231,6 +232,18 @@ def generate_container_mgmt(self, package: Package):
 
         run_opt.append('-t')
 
+        if not is_asic_service:
+            if container_spec['network']:
+                docker_network_type = container_spec['network']
+                run_opt.append(f'--net={docker_network_type}')
+
+        if is_asic_service:
+            if container_spec['network']:
```

Comment:
Isn't the 'network' set to 'host' by default? Think this will always fail if asic-service is True